src/main/kotlin/app/RepoExplorer.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app
+
+import app.utils.CommandExplore
+import java.io.File
+
+/**
+ * RepoExplorer analyzes repositories and uploads stats to server.
+ */
+class RepoExplorer(options: CommandExplore) {
+    val path = options.path
+
+    fun explore() {
+        var isValidPath = false
+
+        try {
+            val f = File(path)
+            isValidPath = f.isDirectory
+        } catch (e: SecurityException) {
+        }
+
+        println(isValidPath)
```

Comment:
it would be better if you use logger,
check this https://github.com/jupf/staticlog